my-dsl.md

```diff
@@ -1,38 +1,60 @@
 # Language
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
-
+The name of the language is EasyLanguage. There is documentation
+here: http://www.tradestation.com/~/media/Files/TradeStation/Education/University/School%20of%20EasyLanguage/Books/EL_Essentials.ashx
 
 # Domain
 _Describe the language's domain in five words._
-
+Financial traders who use TradeStation
 
```

Comment:
I'll be honest, I looked up TradeStation to 1) See if its one or two words :P and 2) check it out since I am interested in programming languages that interact with financial traders.